packages/rancher-istio/charts/values.yaml

```diff
@@ -32,11 +32,11 @@ pilot:
     image: pilot
 
 policy:
-    enabled: true
+    enabled: false
```

Comment:
Why did these all get switched to false ? Egress can stay false, but are policy and telemetry off by default ?

Reply:
I turned them off by default since we don't have some of the other addons  - the demo profile also has them off by default. I went ahead and changed them to true again since we will likely switch them back in the next few prs